### PR TITLE
Added a condition if ESS title is missing

### PR DESCRIPTION
**Pull request by richmadetech**
## Description of change

Some ESS Interactions don't have a title, have replaced these instances with 'ESS Inbound Enquiry' instead. 

## Test instructions

Navigate to One List Corp on Dev, and you will see a list of Export Support Service Interactions, with titles like 'test' or 'ESS Inbound Enquiry'

### Before
Screenshot 2022-12-20 at 13 07 10

### After

Screenshot 2022-12-20 at 13 06 28

## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'

- [ ] Has the branch been rebased to main?
- [ ] Automated tests (Any of the following when applicable: Unit, Functional or End-to-End)
- [ ] Manual compatibility testing (Browsers: Chrome, Firefox, Edge, Safari)

### CODE REVIEW

File: test/sandbox/routes/v4/company/company.js [hunk 1]
```diff
@@ -129,6 +130,7 @@ exports.company = function (req, res) {
     'b319d019-444a-4d2f-9e76-c70f84bb22f6': companyCanadianProvince,
     'c79ba298-106e-4629-aa12-61ec6e2e47ce': companyWithAllActivities,
     '57c41268-26be-4335-a873-557e8b0deb29': companyWithManyContacts,
+    'c79ba298-106e-4629-aa12-61ec6e2e47be': companyWithEssInteractionnoTitle,
```
Comment (chopkinsmade) [inline test/sandbox/routes/v4/company/company.js:133]: companyWithEssInteractionnoTitle -> companyWithEssInteractionNoTitle

**Review by chopkinsmade — APPROVED** [review]

File: test/sandbox/routes/v4/company/company.js (hunk 1 above)
Comment (richmadetech) [reply test/sandbox/routes/v4/company/company.js]: good spot! Cheers

File: test/sandbox/routes/v4/activity-feed/activity-feed.js [hunk 2]
```diff
@@ -290,11 +296,16 @@ exports.activityFeed = function (req, res) {
...
-        : dataHubAndExternalActivities
-    )
+    var activities
+    if (company === BEST_EVER_COMPANY) {
+      activities = companyActivities
+    } else if (company === BEST_EVER_COMPANY_2) {
+      activities = essInteractionsNoTitle
+    } else {
+      activities = dataHubAndExternalActivities
+    }
```
Comment (paulgain) [inline test/sandbox/routes/v4/activity-feed/activity-feed.js:306]: We try not to use `var` favouring `const` instead:
```js
const activities =
  company === BEST_EVER_COMPANY
    ? companyActivities
    : company === BEST_EVER_COMPANY_2
    ? essInteractionsNoTitle
    : dataHubAndExternalActivities
```

Comment (richmadetech) [reply test/sandbox/routes/v4/activity-feed/activity-feed.js]: AH yea good spot, that was clearly old school thinking - forgot you could stack ternary's. - have updated

**Review by paulgain — APPROVED** [review]

**Pull request merged**